### PR TITLE
[zh]: Rsyc content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

### PR DESCRIPTION
**Pull request by zaunist**
…/install-kubeadm.md

part of  #33475  setup-1

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -130,94 +129,111 @@ nc 127.0.0.1 6443
...
+has to be installed. cri-dockerd is a project based on the legacy built-in
+Docker Engine support that was [removed](/dockershim) from the kubelet in version 1.24.
+-->
+{{< note >}}
+Docker Engine 没有实现 [CRI](/zh/docs/concepts/architecture/cri/)，而这是容器运行时在 Kubernetes 中工作所需要的。
+为此，必须安装一个额外的服务 [cri-dockerd](https://github.com/Mirantis/cri-dockerd)。
+cri-dockerd 是一个基于传统的内置Docker引擎支持的项目，它在 1.24 版本从 kubelet 中[移除](/dockershim)。
+
+<!--
+{{< /note >}}
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:190]:
```suggestion
{{< /note >}}
<!--
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 2]
```diff
@@ -130,94 +129,111 @@ nc 127.0.0.1 6443
...
+Docker Engine does not implement the [CRI](/docs/concepts/architecture/cri/)
+which is a requirement for a container runtime to work with Kubernetes.
+For that reason, an additional service [cri-dockerd](https://github.com/Mirantis/cri-dockerd)
+has to be installed. cri-dockerd is a project based on the legacy built-in
+Docker Engine support that was [removed](/dockershim) from the kubelet in version 1.24.
+-->
+{{< note >}}
+Docker Engine 没有实现 [CRI](/zh/docs/concepts/architecture/cri/)，而这是容器运行时在 Kubernetes 中工作所需要的。
+为此，必须安装一个额外的服务 [cri-dockerd](https://github.com/Mirantis/cri-dockerd)。
+cri-dockerd 是一个基于传统的内置Docker引擎支持的项目，它在 1.24 版本从 kubelet 中[移除](/dockershim)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:187]:
```suggestion
cri-dockerd 是一个基于传统的内置Docker引擎支持的项目，它在 1.24 版本从 kubelet 中[移除](/zh/dockershim)。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 3]
```diff
@@ -130,94 +129,111 @@ nc 127.0.0.1 6443
...
+The tables below include the known endpoints for supported operating systems:
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux" %}}
 -->
-默认情况下， kubeadm 使用 {{< glossary_tooltip term_id="docker" >}} 作为容器运行时。
-kubelet 可以使用已弃用的 dockershim 适配器与 Docker Engine 集成（dockershim 是 kubelet 本身的一部分）。
-参阅[容器运行时](/zh/docs/setup/production-environment/container-runtimes/)
-以了解更多信息。
+下面的表格包括支持的操作系统的已知端点。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:197]:
```suggestion
下面的表格包括被支持的操作系统的已知端点。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 4]
```diff
@@ -130,94 +129,111 @@ nc 127.0.0.1 6443
...
+| CRI-O                              | `unix:///var/run/crio/crio.sock`             |
+| Docker Engine (using cri-dockerd)  | `unix:///var/run/cri-dockerd.sock`           |
+{{< /table >}}
+-->
+{{< table >}}
+| 运行时                              | 域套接字                                     |
+|------------------------------------|----------------------------------------------|
+| containerd                         | `unix:///var/run/containerd/containerd.sock` |
+| CRI-O                              | `unix:///var/run/crio/crio.sock`             |
+| Docker Engine (using cri-dockerd)  | `unix:///var/run/cri-dockerd.sock`           |
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:216]:
```suggestion
| Docker Engine (使用 cri-dockerd)  | `unix:///var/run/cri-dockerd.sock`           |
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 5]
```diff
@@ -130,94 +129,111 @@ nc 127.0.0.1 6443
...
+| Docker Engine (using cri-dockerd)  | `npipe:////./pipe/cri-dockerd`               |
+{{< /table >}}
+-->
+{{% tab name="Windows" %}}
+
+{{< table >}}
+| 运行时                              | 域套接字                                     |
+|------------------------------------|----------------------------------------------|
+| containerd                         | `npipe:////./pipe/containerd-containerd`     |
+| Docker Engine (using cri-dockerd)  | `npipe:////./pipe/cri-dockerd`               |
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:235]:
```suggestion
| Docker Engine (使用 cri-dockerd)  | `npipe:////./pipe/cri-dockerd`               |
```

**Comment by zaunist** [comment]
Merge to https://github.com/kubernetes/website/pull/33634, so close this PR.

**Pull request closed without merging**